tests/test_wave_03.py

```diff
@@ -54,13 +54,13 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
-
-    raise Exception("Test needs to be completed.")
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
+    assert INTRIGUE_3 in friends_unique_movies
+    assert FANTASY_4 in friends_unique_movies
```

Comment:
Tests work better when they are comprehensive, so in this case I recommend checking that all of the expected movies are in `friends_unique_movies`:
```suggestion
    assert INTRIGUE_3 in friends_unique_movies
    assert FANTASY_4 in friends_unique_movies
    assert HORROR_1 in friends_unique_movies
```